docs/Upgrade-Guide/Generic_Upgrade_procedure.md

```diff
@@ -24,7 +24,23 @@ This procedure involves upgrading **one server at a time**, while keeping the vo
         
 2. Stop all applications that run on this server and access the volumes via gfapi (qemu, NFS-Ganesha, Samba, etc.)
 
-3. Install Gluster new-version
+3. Install Gluster new-version, below example shows how to create a repository on fedora and use it to upgrade : 
+
+    3.1  Create a private repository (assuming /new-gluster-rpms/  folder has the new rpms ): 
+
+        # createrepo /new-gluster-rpms/
+
+    3.2  Create the .repo file in /etc/yum.d/ : 
+
+        # cat /etc/yum.d/newglusterrepo.repo 
+         [newglusterrepo]
+         name=NewGlusterRepo
+         baseurl="file:///new-gluster-rpms/"
+         gpgcheck=0
```

Comment:
Add one more param in the file `enabled=1` so the complete thing should be as below:
```
        # cat /etc/yum.d/newglusterrepo.repo 
         [newglusterrepo]
         name=NewGlusterRepo
         baseurl="file:///new-gluster-rpms/"
         gpgcheck=0
         enabled=1
```

Reply:
Done